### PR TITLE
Adding automatic location

### PR DESCRIPTION
**Pull request by giubil**
Adding geolocation from browser when latitude and longitude are not specified

### CODE REVIEW

File: src/current-weather.html [hunk 1]
```diff
@@ -226,6 +226,15 @@
           tempLevel: getTempLevel(res.main.temp, this.imperial)
         };
         this.fire('current-weather-load', {weather: this.data});
+      },
+      showLocation: function(position) {
```
Comment (dotch) [inline src/current-weather.html]: I think this should not be called showLocation, but rather getLocation or retrieveLocation or something like that.

**Comment by dotch** [comment]
Thanks, cool! Could you please also add some info to the readme for this?

And maybe this should be an opt-in behavior that can be activated with an attribute, what do you think?

**Comment by giubil** [comment]
I changed to function name and Readme.

To me it seems more logical that when you don't provide latitude and longitude, it should use the geolocation of your browser. Simply because when you send a latitude and longitude, you want to force the position to the user, and when you don't send any, it just means that you let the browser handle it

**Comment by dotch** [comment]
sounds reasonable! Thanks for the PR.

**Pull request merged**